src/components/structures/MatrixChat.js

```diff
@@ -1312,7 +1312,7 @@ export default React.createClass({
         // if the decryption was a failure
         cli.on("Event.decrypted", (e) => {
             if (e.isDecryptionFailure()) {
-                Analytics.trackEvent('E2E', 'Decryption failure');
+                Analytics.trackEvent('E2E', 'Decryption failure', 'ev.content.body: ' + e.getContent().body);
```

Comment:
This commit says one thing but does something completely different.
Is it including the decryption error or the body of the content of the failed to decrypt event?

Reply:
(it looks to be doing the latter but saying it does the former)

Reply:
For better or for worse, we embed the `reason` for the failure in the body of the event once it transforms in to a "Decryption Failure":
```js
    _badEncryptedMessage: function(reason) {
        return {
            clearEvent: {
                type: "m.room.message",
                content: {
                    msgtype: "m.bad.encrypted",
                    body: "** Unable to decrypt: " + reason + " **",
                },
            },
        };
    },
```

So it *looks* like it's doing the wrong thing but it's not. 

Maybe needs a comment?

Reply:
oh yeah thats not pretty :(
comment would help